### ISSUE TITLE
Plugin 52 Datastore Consistency v1.6 repeats datastore name

### ISSUE THREAD

**Issue by spartyzik** [alanrenouf/vCheck-vSphere#672]
I get the datastore name repeated per host:

```
Name	                        Cluster
datastore_01 datastore_01	cluster-01
```

Line 23
`         Select-Object @{Name="Name"; Expression={$_.Group.name}}, @{Name="Cluster";Expression={$Cluster.Name}}`

should be

`         Select-Object @{Name="Name"; Expression={$_.Name}}, @{Name="Cluster";Expression={$Cluster.Name}}`

**Comment by jm-73** [comment]
This project seems to be kind of dead. I have sent mail to the author that I am willing to contribute as I have found some bugs. I have also made some minor improvements to some plugins. But its not to any good use if nobody accepts the issues and the pull requests. :-(

**Comment by vScripter** [comment]
@jm-73 Hi Jens, I'm Kevin (one of the maintainers; I think there are 4-5 of us). I can't disagree with your comment and I've actually been looking through some of the issues and PRs that have been sitting out there, in hopes that I can pick a point-in-time which we can 'start fresh', in a way. There are a number of challenges to maintaining all the pieces and parts and I'd like to get some development standards defined, which will make it easier to maintain. 

Are you a member of the [VMware {code} Slack](https://code.vmware.com/join)?

**Comment by jm-73** [comment]
Hi vScripter! Nice to see that there are some hope for this project. No I am not a member of VMware Code Slack (I just requested to join). I have read that Alan said that there are some slack collaboration. But I couldn't figure out how to enter. Thanks for the link!
I think that there are a lot of potential in the current code. I am not a very experienced coder , but I can see that a lot of the plugins are not following the same standard. 
I would really like to help out in any way I can. Mainly because vCheck has helped me many years as an VMware senior expert.

**Comment by alanrenouf** [comment]
Yeah I really need to dig back in there as well, this project is still loved by many people and i havent been able to spend the time on it recently (well years), definitely open for more help, also need to get some automated testing going on to make it easier to accept some of the easier changes!

**Comment by alanrenouf** [comment]
If you are chatting on Slack then mention me so I can come join in, hopefully i can spend a little time on this again now
